README.md

```diff
@@ -84,48 +84,34 @@ $ kubectl delete -f example/example-etcd-cluster.yaml
 
 ## Resize an etcd cluster
 
-`kubectl apply` doesn't work for TPR at the moment. See [kubernetes/#29542](https://github.com/kubernetes/kubernetes/issues/29542).
-As a workaround, we use cURL to resize the cluster.
+**Note**: In order to use `kubectl apply` you will need Kubernetes 1.6 or higher. Otherwise you will have to post the changes to the API server directly using `curl`.
 
 Create an etcd cluster:
 
 ```
-$ kubectl create -f example/example-etcd-cluster.yaml
+$ kubectl create -f example/example-etcd-cluster.yaml --save-config
```

Comment:
This is to suppress the warning that would otherwise come without this flag:
```
Warning: kubectl apply should be used on resource created by either kubectl create --save-config or kubectl apply
```
Although all the `save--config` flag does is save the configuration in an annotation field.
```
apiVersion: v1
items:
- apiVersion: etcd.coreos.com/v1beta1
  kind: Cluster
  metadata:
    annotations:
      kubectl.kubernetes.io/last-applied-configuration: |
        {"apiVersion":"etcd.coreos.com/v1beta1","kind":"Cluster","metadata":{"annotation
s":{},"name":"example-etcd-cluster","namespace":"e2e-local"},"spec":{"size":5,"version":
"3.1.4"}}
. . .
```